### ISSUE TITLE
After upgrading to new CLI and NPM I get error on ionic cordova resoruces 

### ISSUE THREAD

**Issue by CreativeArtDesign** [ionic-team/ionic-cli#3840]
**Description:**

After upgraing i getting this error when adding a project or building resourses.

**ionic cordova resoruces** 
For more information 
Spash screen works but not for icons

 ionic cordova resoruces -i
I have an icon and updated it in 1024x1024 but it say no icon found

[ERROR] Source image files were not found for the following platforms/types:
        
        - ios/icon**Steps to Reproduce:**
**Output:**

**My `ionic info`:**

```
Ionic:

   ionic (Ionic CLI)  : 4.10.1 (/usr/local/lib/node_modules/ionic)
   Ionic Framework    : ionic-angular 3.9.2
   @ionic/app-scripts : 3.2.1

Cordova:

   cordova (Cordova CLI) : 8.1.2 (cordova-lib@8.1.1)
   Cordova Platforms     : ios 4.5.5
   Cordova Plugins       : cordova-plugin-ionic-keyboard 2.1.3, cordova-plugin-ionic-webview 2.3.1, (and 7 other plugins)

System:

   NodeJS : v10.15.0 (/usr/local/bin/node)
   npm    : 6.7.0
   OS     : macOS Mojave
   Xcode  : Xcode 10.1 Build version 10B61

```

**Other Information:**

**Comment by CreativeArtDesign** [comment]
I think I found the problem, I created an icon In illustrator 2019 and exported the icon in 1024x1024 to PNG file then. Ioinic did not find the file. Tried to remove the Alfa channel and same not working.

So I spend an old icon.png that Is shipped in ionic and then I opened it in photoshop and put my image. Then saved it as a png. Then it worked.

So the issue is when I export it from illustrator it does not found it.

What must the preferences be for this icon file? PNG? Transparent OFF? What?



Best Regards


CREATIVE ART DESIGN
Global Services
Mr. Kamal S.M.M. Ahmad
E-Mail
cad.co@yahoo.com
